File: tests/fixtures/py/sample.py
# Sample Python file for integration tests

def greet(name):
    """Greet a person by name"""
    print(f"Hello, {name}")
    return f"Greeting sent to {name}"

def calculate(x, y):
    """Calculate sum of two numbers"""
    result = x + y
    return result

def process_data(data):
    """Process data and return results"""
    return [item * 2 for item in data]

class DataProcessor:
    def __init__(self, name):
        self.name = name
    
    def process(self, data):
        return self.name + str(data)

File: tests/fixtures/rs/sample.rs
// Sample Rust file for integration tests

fn add(a: i32, b: i32) -> i32 {
    a + b
}

fn multiply(x: i32, y: i32) -> i32 {
    x * y
}

fn calculate_sum(numbers: &[i32]) -> i32 {
    numbers.iter().sum()
}

fn main() {
    println!("Hello, world!");
    let result = add(5, 3);
    println!("Result: {}", result);
}

struct Point {
    x: i32,
    y: i32,
}

impl Point {
    fn new(x: i32, y: i32) -> Point {
        Point { x, y }
    }
}
